Documentation/api.md

```diff
@@ -146,7 +146,7 @@ The `ClusterMonitoringConfiguration` resource defines settings that customize th
 
 #### Description
 
-The `K8sPrometheusAdapter` resource defines settings for the Prometheus Adapter component.
+The `K8sPrometheusAdapter` resource defines settings for the Prometheus Adapter component. This is deprecated and will be removed in a future version, setting it has no effect.
```

Comment:
The `K8sPrometheusAdapter` resource is removed since 4.16 after metrics-server GA